pages/1.11/release-notes/1.11.1/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.1
+title: Release Notes for 1.11.1
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.11.1.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.0/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
```

Comment:
Change version to 1.11.1

Reply:
Yup!  `https://downloads.dcos.io/dcos/stable/1.11.1/dcos_generate_config.sh` here.

Reply:
Done.